### PR TITLE
Rename SkolemFunId to SkolemId

### PR DESCRIPTION
**Pull request by ajreynol**

### CODE REVIEW

File: include/cvc5/c/cvc5.h [hunk 1]
```diff
@@ -1506,15 +1506,15 @@ bool cvc5_term_is_skolem(Cvc5Term term);
...
+Cvc5SkolemId cvc5_term_get_skolem_id(Cvc5Term term);
 /**
  * Get the skolem indices of a term.
  * @note Asserts isSkolem().
  * @warning This function is experimental and may change in future versions.
  * @param term The skolem.
  * @return The skolem indices of the term. This is list of terms that the
  * skolem function is indexed by. For example, the array diff skolem
- * `SkolemFunId::ARRAY_DEQ_DIFF` is indexed by two arrays.
+ * `SkolemId::ARRAY_DEQ_DIFF` is indexed by two arrays.
```
Comment (abdoo8080) [inline include/cvc5/c/cvc5.h:1517]:
```suggestion
 * `Cvc5SkolemId::ARRAY_DEQ_DIFF` is indexed by two arrays.
```

File: src/api/python/cvc5.pxi [hunk 2]
```diff
@@ -5460,7 +5460,7 @@ cdef class Term:
 
            :return: The skolem indices of this term. This is list of terms that
             the skolem function is indexed by. For example, the array diff
-            skolem `SkolemFunId::ARRAY_DEQ_DIFF` is indexed by two arrays.
+            skolem `SkolemId::ARRAY_DEQ_DIFF` is indexed by two arrays.
```
Comment (abdoo8080) [inline src/api/python/cvc5.pxi:5463]:
```suggestion
            skolem `SkolemId.ARRAY_DEQ_DIFF` is indexed by two arrays.
```

File: src/theory/strings/skolem_cache.h [hunk 3]
```diff
@@ -187,17 +188,17 @@ class SkolemCache
...
   /** the optional rewriter */
   Rewriter* d_rr;
   /** string type */
   TypeNode d_strType;
   /** Constant node zero */
   Node d_zero;
   /** map from node pairs and identifiers to skolems */
-  std::map<Node, std::map<Node, std::map<SkolemId, Node> > > d_skolemCache;
+  std::map<Node, std::map<Node, std::map<StringSkolemId, Node> > >
+      d_skolemCache;
```
Comment (abdoo8080) [inline src/theory/strings/skolem_cache.h:201]: Format

**Review by abdoo8080 — APPROVED** [review]

File: src/theory/strings/skolem_cache.h (hunk 3 above)
Comment (ajreynol) [reply src/theory/strings/skolem_cache.h:201]: This is formatted, apparently.

**Pull request merged**